pkg/localstore/subscription_push.go

```diff
@@ -52,14 +53,15 @@ func (db *DB) SubscribePush(ctx context.Context) (c <-chan swarm.Chunk, stop fun
 		// close the returned chunkInfo channel at the end to
 		// signal that the subscription is done
 		defer close(chunks)
-		// sinceItem is the Item from which the next iteration
-		// should start. The first iteration starts from the first Item.
-		var sinceItem *shed.Item
+		// lastItem is the first Item received in the last iteration.
+		var lastItem *shed.Item
+		// toItemKey is the key for the Item that was oldest in the last iteration.
```

Comment:
what does this mean? if `lastItem` is the first item that was received in the last iteration, then it is also the oldest item in the last iteration, and then why the differentiation calling one `last` and one `to`